piqe_ocp_lib/tests/resources/test_ocp_operators.py

```diff
@@ -126,7 +126,17 @@ def test_get_package_singlenamespace_channel(self, get_test_objects):
         else:
             logger.warning("The randomly picked package doesn't seem to have a single namespace channel")
 
-
+    
+    def test_get_crd_models_from_manifest(self, get_test_objects):
+        cmfm_obj = get_test_objects.op_hub_obj
+        alm_list=cmfm_obj.get_crd_models_from_manifest('amq-streams', 'stable')
+        assert len(alm_list)!= 0
+        for i in range(0,len(alm_list)):
```

Comment:
May be a nitpic, I don't know if 'api_version' and 'kind' will always have the same index in the list. It's probably better to just iterate ( for e in alm_list) then just check if those strings are in alm_list.keys()